src/main/java/seedu/mindmymoney/helper/Calculations.java

```diff
@@ -102,8 +102,8 @@ public static String printBar(float percentage) {
     /**
      * Calculates the percentage of expenses for a particular category type.
      *
-     * @param categoryType Category type to calculate for.
-     * @param foundItems The list containing all the expenses in the month.
+     * @param categoryType     Category type to calculate for.
+     * @param foundItems       The list containing all the expenses in the month.
```

Comment:
Not too sure what this block is, but if its a white space perhaps can remove.

Reply:
its what i auto format the code, ok will remove